continuous_integration/recipe/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev') %}
+{% set nightly_version = version + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+
+package:
+  name: dask
+  version: {{ nightly_version }}
+
+source:
+  git_url: ../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  noarch: python
+  string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.7
+  run:
+    - python >=3.7
+    - dask-core {{ version }}
+    - distributed {{ version }}
```

Comment:
Right now we are pinning runtime dependencies of `dask-core` and `distributed` to their latest stable versions:

- should we opt to build and upload `dask-core` as well so we can pin that dependency to a nightly version?
- generally speaking, is it reasonable to run distributed with a newer version of dask?

Reply:
Yeah that's what I'm thinking at least from a practical standpoint